swagger.yaml

```diff
@@ -6210,6 +6210,7 @@ definitions:
     required:
       - openshift_version
       - cpu_architecture
+      - cpu_architectures
```

Comment:
@danielerez I wonder about this part, what do you think?

Reply:
I think adding a new required field is a breaking API change.

Reply:
ACK, removing the requiredness right now

Reply:
Right, should be optional for now. Should be probably handled when removing cpu_architecture later on, hope it won't be too much mess:(

Reply:
Reverted. Yeah, in the future we should just swap which one is required